console/console-init/ui/src/modules/connection-detail/containers/ConnectionDetailContainer.tsx

```diff
@@ -27,7 +30,7 @@ interface IConnectionLinksListPageProps {
   sortValue?: ISortBy;
   setSortValue: (value: ISortBy) => void;
 }
-export const ConnectionLinksListPage: React.FunctionComponent<IConnectionLinksListPageProps> = ({
+export const ConnectionDetailContainer: React.FunctionComponent<IConnectionDetailContainerProps> = ({
```

Comment:
I think ConnnectionLinksContainer is better than ConnectionDetailContainer as it deals with links of single connection